src/main/java/Sun.java

```diff
@@ -0,0 +1,93 @@
+        do{
+            command = scanner.nextLine();
+
+            if (!command.equals("bye")) {
+                if (command.equals("list")) {
+                    System.out.println("____________________________________________________________");
+                    System.out.println("Here are the tasks in your list:");
+                    for (int i = 0; i < taskCount; i++) {
+                        System.out.println((i + 1) + "." + tasks[i].getType()
+                                +  tasks[i].getStatus() + " "+ tasks[i].description);
+                    }
+                    System.out.println("____________________________________________________________");
+                } else if (command.startsWith("mark ")) {
+                    int taskIndex = Integer.parseInt(command.substring(5)) - 1;
+                    if (taskIndex >= 0 && taskIndex < taskCount) {
+                        tasks[taskIndex].markAsDone();
+                        System.out.println("____________________________________________________________");
+                        System.out.println("Nice! I've marked this task as done:");
+                        System.out.println("  " + tasks[taskIndex].getType() +
+                                tasks[taskIndex].getStatus() + " " + tasks[taskIndex].description);
+                        System.out.println("____________________________________________________________");
+                    }
+                } else if (command.startsWith("unmark ")) {
+                    int taskIndex = Integer.parseInt(command.substring(7)) - 1;
+                    if (taskIndex >= 0 && taskIndex < taskCount) {
+                        tasks[taskIndex].markAsNotDone();
+                        System.out.println("____________________________________________________________");
+                        System.out.println("OK, I've marked this task as not done yet:");
+                        System.out.println("  " + tasks[taskIndex].getType() +
+                                tasks[taskIndex].getStatus() + " " + tasks[taskIndex].description);
+                        System.out.println("____________________________________________________________");
+                    }
+                } else if (command.startsWith("todo ")){
+                    Todo todoTask = new Todo(command);
+                    tasks[taskCount] = todoTask;
+                    taskCount++;
+                    System.out.println("____________________________________________________________");
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println(todoTask.getType() +  todoTask.getStatus() + " "+ todoTask.description);
+                    System.out.println("Now you have " + taskCount + " tasks in the list.");
+                    System.out.println("____________________________________________________________");
+                }else if (command.startsWith("deadline ")){
```

Comment:
Maybe a space after } would look neater
```suggestion
                } else if (command.startsWith("deadline ")){
```